### ISSUE TITLE
[MAS3.2.4] Meaningful sequence is not correct at 'Depart and Arrive' text content while navigating in scan mode.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1351]
**Actual Result:**
When focus lands on the Depart and Arrives text, narrator is not reading whole content sequentially.
Meaningful sequence is not correct at ''Depart and Arrive' text content while navigating in scan mode.

**Note** - Same issue repro at - 
1. URL -> Help -> Show Flight Update using  Adaptive card button -> New uploaded data -> Passengers and Seat - Narrator is not reading whole content sequentially. Attachment - "MAS1.3.2_Meaningful sequence  at Passengers and Seat".
2. URL -> Help -> Show Weather Forecast button -> New uploaded data ->Sat. Sun. Mon.  and Seattle, WA text - Narrator is not reading whole content sequentially.Attachment - "MAS1.3.2_Meaningful sequence at Text under Weather Forecast content ".
3. URL -> Help -> Receipt card Button ->  New uploaded data ->Surface Pro4’ content - Narrator is not reading whole content sequentially.Attachment - **"MAS1.3.2_SurfacePro4"**.

**Expected Result:**
Meaningful sequence should be correct at 'Depart and Arrive' text content while navigating in scan mode.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4.  Content will upload, Navigate to any "Show Flight Track using  Adaptive card " button by using Tab key and press Enter on it.
5. New data will upload.
6. Open Narrator with Scan mode (Caps+Space key)
7. Navigate to 'Depart and Arrive' text content by using Up/Down arrow keys.
8. Observe that meaningful sequence is not correct at 'Depart and Arrive' text content while navigating in scan mode.

**User Impact:**
If focus not lands on text in sequential way then user will get confuse and not able to understand the content and not able to perform the action.

**MAS Reference:**
[MAS 1.3.2 - Meaningful Sequence (25)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
1. [MAS1.3.2_Meaningful sequencing is not correct while navigate in scan mode..zip](https://github.com/Microsoft/BotFramework-WebChat/files/2580071/MAS1.3.2_Meaningful.sequencing.is.not.correct.while.navigate.in.scan.mode.zip)
2. [MAS1.3.2_Meaningful sequence  at Passengers and Seat.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2580095/MAS1.3.2_Meaningful.sequence.at.Passengers.and.Seat.zip)
3. [MAS1.3.2_Meaningful sequence at Text under Weather Forecast content.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2580441/MAS1.3.2_Meaningful.sequence.at.Text.under.Weather.Forecast.content.zip)
4. [MAS1.3.2_SurfacePro4.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2588302/MAS1.3.2_SurfacePro4.zip)

**Comment by compulim** [comment]
@corinagum see if we can fix it in our MockBot, to use a single column container for the text.

**Comment by corinagum** [comment]
Investigating either as bot-specific, or part of Adaptive Cards

**Comment by compulim** [comment]
Adaptive Card team please look at the nested button issues (3-nested now).

**Comment by corinagum** [comment]
Adaptive Cards issue can be found [here](https://github.com/Microsoft/AdaptiveCards/issues/2241).

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG3.2.4

**Comment by RebeccaAnne** [comment]
This is now tracked by https://github.com/microsoft/AdaptiveCards/issues/4815
